### PR TITLE
LPS-143632 Remove LanguageUtil from moduels part 3a mixed

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:acceptance

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
Could you please help me scan this test result?
Thank you!

**Comment by catarineaguiar** [comment]
ci:test:acceptance

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/AddFormInstanceRecordMVCResourceCommandTest.java [hunk 1]
```diff
@@ -168,6 +163,20 @@ private static void _setUpDDMFormInstance() throws Exception {
...
+
+	private static void _setUpLanguageUtil() {
+		LanguageUtil languageUtil = new LanguageUtil();
+
+		languageUtil.setLanguage(_language);
+
+		ReflectionTestUtil.setFieldValue(
+			_addFormInstanceRecordMVCResourceCommand, "_language", _language);
+	}
+
```
Comment (tinatian) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/AddFormInstanceRecordMVCResourceCommandTest.java:179]: @julschong Duplicate setting, please fix.

**Comment by tinatian** [comment]
@julschong Please see my comment and review your fix for unit tests again.

**Pull request closed without merging**